src/Helper/DisqusHelper.php

```diff
@@ -44,16 +41,11 @@ public function render($name, array $parameters = array(), $template = 'KnpDisqu
 
```

Comment:
Above, we should catch `TransportExceptionInterface` now (because we should allow this to be thrown from the `Disqus` class)